easybuild/easyconfigs/m/MaSuRCA/MaSuRCA-3.2.2-foss-2016a.eb

```diff
@@ -0,0 +1,44 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+#
+# Copyright:: Copyright 2017 University of Geneva
+# Authors::   Yann Sagon <yann.sagon@unige.ch>
+# License::   MIT/GPL
+# $Id$
+#
+##
+
+easyblock = 'ConfigureMake'
+
+name = 'MaSuRCA'
+version = '3.2.2'
+
+homepage = 'http://www.genome.umd.edu/masurca.html'
+
+description = '''MaSuRCA is whole genome assembly software. It combines the efficiency of the de Bruijn graph  
+ and Overlap-Layout-Consensus (OLC) approaches. MaSuRCA can assemble data sets containing 
+ only short reads from Illumina sequencing or a mixture of short reads and long reads 
+ (Sanger, 454, Pacbio and Nanopore).'''
+
+# need a temporary url to download it. Do it manually here:
+# http://www.genome.umd.edu/masurca_form.html
+sources = ['%(name)s-%(version)s.tar.gz']
+
+dependencies = [
+    ('libreadline', '6.3'),
+    ('Tcl', '8.6.5'),
+    ('Boost', '1.61.0'),
+    ('zlib', '1.2.8'),
+]
+
+toolchain = {'name': 'foss', 'version': '2016a'}
```

Comment:
Sorry if I wasn't clear, this should stay up (above `sources`) style-wise...

We hope to start automatic style checks soon, including order of parameters.